### ISSUE TITLE
device orientation issue

### ISSUE THREAD

**Issue by moovida** [dsandler/markers#2]
I am not sure about the reason of keeping a forced orientation in the config files, so I will just comment my issue based on not-knowing. :)

the problem is that on all my devices, Galaxy Nexus and Nexus 7, the orientation forces to portrait. While I understand that on the phone portrait might be better for the color frame layout, I guess that on the tab it should be possible to choose landscape. Or better, it should appear in landscape if the device is in that orientation.

**Comment by GitHubDroid** [comment]
Hey andrea,

I am having a similar issue where my Panasonic Toughpad FZ-A1 is only opening in landscape, and will not go to portrait at all.

**Comment by dsandler** [comment]
Yeah, this is WAI (working as intended) for now. What I'd like to do is have Markers rotate the UI around the canvas, but I haven't gotten around to implementing that yet. (One of the things that's tricky in this case is exactly what size to make the canvas, since when the device rotates the space available to the application's content changes a bit, and Markers has traditionally sized its canvas based on the content rect.)

**Comment by AlexanderMadani** [comment]
Hi Guys, 

   I am relatively new to this community (and Android Development in general).  But I must say congratulations to Mr. Daniel Sandler for this amazing project ! 

   Okay so I have a question.  I am trying to incorporate Markers into my App, and I am having a hell of a time because its my first time doing such a thing.  I downloaded the Zip file, and in Android Studio, the Import was fairly easy (Studio did most of the work for me).  So now I see that there is an "app" folder followed by a "Gradle Scripts" folder.   I have a Paint button on my MainActivity for my App, and I would like to invoke "Markers" once the user presses on that "Paint" button.

  Can any of you dear Devs and Android Fans here help me figure out how to successfully (and hopefully painlessly import this Markers project into my own project and then invoke the MainActivity of "Markers" ... which I believe is called (MarkersActivity).

   I am able to run the Markers app successfully now that it has been imported from the Zip file on my desktop.  But I need to figure out how to implement it in my own project, and since there are so many java files and resources involved, I could use a little help.   I tried already to copy and paste everything into my own project but that did not turn out to be as straightforward as I had hoped.

Thanks 
Alex

**Comment by dgorrd** [comment]
The drawing view area is scrollable and resizable even if the canvas is set (one of the nice features).  The view area could simply be resized on rotation (letterboxed to start with) without changing the actual bitmap.  From there the user could pan or scale as needed.
